plugins/lang-groovy/src/test/java/org/elasticsearch/messy/tests/GeoDistanceTests.java

```diff
@@ -222,12 +232,15 @@ public void testSimpleDistance() throws Exception {
     }
 
     public void testDistanceSortingMVFields() throws Exception {
+        Version version = VersionUtils.randomVersionBetween(random(), Version.V_1_0_0, Version.CURRENT);
```

Comment:
I'm a little concerned about all of these "choose a random version" because it will severely reduce the test coverage for the new code? If we are concerned about backcompat, lets have dedicated backcompat tests for pre 2.2 behavior being maintained...